vendor/yubico/u2flib-server/src/u2flib_server/U2F.php

```diff
@@ -222,6 +222,7 @@ public function doRegister($request, $response, $includeCert = true)
     public function getAuthenticateData(array $registrations)
     {
         $sigs = array();
+        $challenge = $this->createChallenge();
```

Comment:
Makes sense!

However, we shoud try to get this merged in the upstream repo https://github.com/Yubico/php-u2flib-server ;-)